akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/internal/ActorSystemStub.scala

```diff
@@ -49,9 +48,9 @@ import akka.annotation.InternalApi
   override val settings: Settings = {
     val classLoader = getClass.getClassLoader
     val dynamicAccess = new ReflectiveDynamicAccess(classLoader)
-    val config =
-      classic.ActorSystem.Settings.amendSlf4jConfig(ConfigFactory.defaultReference(classLoader), dynamicAccess)
-    val untypedSettings = new classic.ActorSystem.Settings(classLoader, config, name)
+    val config_ =
+      classic.ActorSystem.Settings.amendSlf4jConfig(config, dynamicAccess)
+    val untypedSettings = new classic.ActorSystem.Settings(classLoader, config_, name)
```

Comment:
`config_` is a strange name. We sometimes use `_` as a prefix but not as suffix. Here it could be something descriptive as `configWithSlf4j`

Reply:
feel free to follow up if you like, but not important